### PR TITLE
Support for new datatype BigDecimal, BigInteger and UUID

### PR DESCRIPTION
**Pull request by xeonn**
Hi,
I have created converter for the above mentioned datatype based on the following issues reported.

https://github.com/canoo/dolphin-platform/issues/326

Do have a look.

Regards,

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/334)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 group = com.canoo.dolphin-platform
-version = 0.9.0-SNAPSHOT
+version = 0.8.10
```
Comment (hendrikebbers) [inline gradle.properties]: please remove this change. The version will be updated when we do a release in a release branch. The master should be always snapshot

Comment (xeonn) [inline gradle.properties]: How do I do that? Via new commit? or restructuring the commit?

Sorry, this is my first time creating a pull request in github. Was thinking u are going to cherry-pick the 3 bottom commit.

Comment (hendrikebbers) [inline gradle.properties]: you can simply change it in your branch "xeonn:new-datatype". I think the PR should be updated automatically ;)

Comment (xeonn) [inline gradle.properties]: Ok done.

**Pull request merged**